packages/expo-cli/src/commands/utils/validateApplicationId.ts

```diff
@@ -33,8 +33,7 @@ export async function getBundleIdWarningAsync(bundleId: string): Promise<string
 
   const url = `http://itunes.apple.com/lookup?bundleId=${bundleId}`;
   try {
-    const response = await got(url);
-    const json = JSON.parse(response.body?.trim());
+    const json = await fetch(url).then(value => value.json());
```

Comment:
I'm not sure if this is a common pattern throughout the codebase, but I'm not a fan of mixing async/await with then callbacks. I would prefer (even with the extra variable assignment):
```js
const response = await fetch(url)
const json = await response.json()
```

Reply:
```suggestion
    const response = await fetch(url);
    const json = await response.json();
```